tests/unit/badge/badge.spec.js

```diff
@@ -88,4 +88,44 @@ describe('Badge', () => {
 
     expect(wrapper).toMatchSnapshot();
   });
+
+  it('should render badge with "start" alignment', () => {
```

Comment:
Should we have a combined (position + align) test?

Reply:
@leandrooriente good idea, will add now